material-popup-menu/src/main/java/androidx/appcompat/widget/MaterialRecyclerViewPopupWindow.kt

```diff
@@ -105,6 +111,8 @@ class MaterialRecyclerViewPopupWindow(
 
         dropDownHorizontalOffset = a.getDimensionPixelOffset(androidx.appcompat.R.styleable.ListPopupWindow_android_dropDownHorizontalOffset, 0)
         dropDownVerticalOffset = a.getDimensionPixelOffset(androidx.appcompat.R.styleable.ListPopupWindow_android_dropDownVerticalOffset, 0)
+        backgroundDimEnabled = a.getBoolean(R.styleable.ListPopupWindow_android_backgroundDimEnabled, false)
+        backgroundDimAmount = a.getFloat(R.styleable.ListPopupWindow_android_backgroundDimAmount, 0.3f)
```

Comment:
Could you move `0.3f` to a constant inside the `companion object` e.g. `private const val DEFAULT_BACKGROUND_DIM_AMOUNT = 0.3f`?

Reply:
Done.